ADDONS.md

```diff
@@ -46,13 +46,11 @@ Foreground and background colors do not replace one another but they sum up, whi
 
 So the following sequence, which is not terminated by reset, `\u001B[41m``AAA``\u001B[40m``\u001B[37m``BBB` produces as output:
 
-<span style="color:white;background-color:black; display:inline-block; width:100%"><span style="color:black;background-color:red;display:inline-block">AAA</span>BBB    
-</span>
+![withoutreset](colored-message-without-reseting.png "")
 
 While the following sequence, which includes the reset control character, `\u001B[41m``AAA``\u001B[40m``\u001B[37m``BBB``\u001B[0m` produces as output:
 
-<span style="color:black;background-color:red;display:inline-block">AAA</span><span style="color:white;background-color:black;display:inline-block">BBB</br>
-</span>
+![withreset](colored-message-with-reseting.png "")
```

Comment:
I like the idea of using images, but this output is not 100% correct. The '`' characters belong to the Markdown format and not the output. The correct image is:
<img width="858" alt="Screen Shot 2020-06-25 at 20 39 54" src="https://user-images.githubusercontent.com/1557512/85779831-4c001180-b724-11ea-997d-89b6ebf7060e.png">